apps/webapp/src/app/(Magic-Link)/magic-link/page.tsx

```
@@ -0,0 +1,614 @@
+      CONNECTORS_METADATA[selectedProvider.category][selectedProvider.provider];
+
+    if (providerMetadata.authStrategy.strategy === AuthStrategy.oauth2) {
+      setAdditionalParams(values);
+      setStartFlow(true);
+    } else {
+      createApiKeyConnection(
+        {
+          query: {
+            linkedUserId: magicLink?.id_linked_user as string,
+            projectId: projectId,
+            providerName: selectedProvider?.provider!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)